css-typed-om/Overview.bs

```diff
@@ -2847,10 +2847,49 @@ of serializations of the contained {{CSSTransformComponent}} objects.
 
 {{CSSFontFaceValue}} Serialization {#fontfacevalue-serialization}
 -----------------------------------------------------------------
-
+ 
 {{CSSFontFaceValue}} objects are serialized to the value of their contained
 {{CSSFontFaceValue/fontFamilyName}}.
 
+Units and Values Serialization {#unitsandvalues-serialization}
+--------------------------------------------------------------
+
+This section covers how to serialize the various unit types in CSSOM
+
+<div algorithm="serialize a 'length' value">
+
+To <dfn>serialize a 'length' value</dfn>, run these steps:
+    1. Let |values| initially be the empty [=list=].
+    2. If the <length> value has a '-' sign character append it to |values|.
+    3. Append the result of [=serialize a <length> value=] to |values|. 
```

Comment:
What is this algorithm you're invoking? It looks like it's the same algorithm you're defining.  Maybe you meant to serialize the length's value as a number by invoking the "serialize a number" algorithm? (And if so, you want to keep the negative on there, as the number serializer will handle it for you.)